packages/lit-html/src/test/directives/style-map_test.ts

```diff
@@ -12,9 +12,9 @@ import {assert} from '@esm-bundle/chai';
 const ua = window.navigator.userAgent;
 const isChrome41 = ua.indexOf('Chrome/41') > 0;
 const isIE = ua.indexOf('Trident/') > 0;
-const supportsCSSVariables = isIE || isChrome41;
+const supportsCSSVariables = !isIE && !isChrome41;
 const testIfSupportsCSSVariables = (test: any) =>
-  supportsCSSVariables ? test.skip : test;
+  supportsCSSVariables ? test : test.skip;
```

Comment:
This seems like it shouldn't be a change because you're inverting the condition _and_ branches. What am I missing here?

Though I agree this is more correct and oops on that condition!!

Reply:
Originally this used to be
```ts
const testIfSupportsCSSVariables = (test: any) =>
  isIE || isChrome41 ? test.skip : test;
```

Then we got
```ts
const supportsCSSVariables = isIE || isChrome41; // this is actually NOT supports

const testIfSupportsCSSVariables = (test: any) =>
  supportsCSSVariables ? test.skip : test; // we actually want to not skip if supports
```
which semantically was wrong, but ended up skipping the right tests. This particular branching does not change in this PR but does make it semantically correct.

The problematic lines ended up being here:
https://github.com/lit/lit/blob/8a11f7ea0964c8d8f055e1a085e3f5b99877dccd/packages/lit-html/src/test/directives/style-map_test.ts#L71-L74
where those assertions were actually not skipping for IE, so the change in this PR makes sure these assertions are skipped.

Reply:
ah... thanks for the explanation!